ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -232,10 +237,10 @@ public Response handleKsqlStatements(
       return Errors.badStatement(e.getRawMessage(), e.getSqlStatement());
     } catch (final KsqlException e) {
       return ErrorResponseUtil.generateResponse(
-          e, Errors.badRequest(e));
+          e, Errors.badRequest(e), errorHandler);
```

Comment:
Feels wrong to be passing and instance of `Errors` back to a static method of `Errors`.

Maybe change `badRequest` to being a member, rather than static, function?

Same for others.